packages/form/src/form.vue

```diff
@@ -11,15 +11,9 @@
 </template>
 
 <script lang="ts">
-import {
-  defineComponent, provide, watch, ref,
-  computed, reactive, toRefs, PropType,
-} from 'vue'
+import { computed, defineComponent, PropType, provide, reactive, ref, toRefs, watch } from 'vue'
 import mitt from 'mitt'
-import {
-  elFormKey, ElFormItemContext as FormItemCtx,
-  elFormEvents, ValidateFieldCallback,
-} from './token'
+import { elFormEvents, ElFormItemContext as FormItemCtx, elFormKey, ValidateFieldCallback } from './token'
 import { FieldErrorList } from 'async-validator'
```

Comment:
Minor here, this was not your case to fix but I'd like here before the imported member to have a `type` keyword.